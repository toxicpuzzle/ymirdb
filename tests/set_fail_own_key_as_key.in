SET cycle cycle
SET a a
SET b b
SET c 1 2 3
SET a c b a
LIST ENTRIES
bye