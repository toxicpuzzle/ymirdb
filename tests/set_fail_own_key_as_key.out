> not permitted

> not permitted

> not permitted

> ok

> no such key

> c [1 2 3]

> bye
